src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
+  }
+
+  return days * 40;
```

Comment:
The function and variable names should be descriptive. Consider renaming the function to reflect that it's specifically for car rentals and the parameter 'days' to 'numberOfDays' for clarity.